core/types/block.go

```diff
@@ -902,3 +902,16 @@ func (m BlockManifest) EncodeIndex(i int, w *bytes.Buffer) {
 func (m BlockManifest) Size() common.StorageSize {
 	return common.StorageSize(m.Len() * common.HashLength)
 }
+
+// pendingBodyKey comprises all root fields of the header
+type PendingBodyKey struct {
```

Comment:
this is not the key. This is the preimage data to construct the key

Reply:
It called it a day